Gemfile

```diff
@@ -64,6 +62,7 @@ group :development, :test do
   gem 'pry-stack_explorer'
   gem 'rspec-collection_matchers'
   gem 'rspec-rails'
+  gem 'puma'
```

Comment:
this seems weird, we should just switch everything to puma I'd think not just dev, want to do that in a separate PR?  @markottaviani are there any gotchas with our host for puma, it does seem to be the new hotness.

Reply:
I don't know, I only added this because Capybara was complaining about about puma missing when I was running the rspec suite.    Specific error was:

```
Capybara is unable to load `puma` for its server, please add `puma` to your project or specify a different server via something like `Capybara.server = :webrick`
```